src/new_hope/week1/day5.clj

```diff
@@ -0,0 +1,85 @@
+(ns new-hope.week1.day5
+  (:require [midje.sweet :refer :all]
+            [criterium.core :as crit]))
+
+;Write	a	function	which	removes	consecutive	duplicates	from	a	sequence.
+(defn de-dups-helper [st out]
+  (let [h (first st)                                        ;; is there more ideomatic solution? such as h :: t => bla-bla
+        t (rest st)]
+    (cond
+      (= h nil) out
+      (= h (first out)) (recur t out)
+      :else (recur (rest st) (cons h out)))))
+
+(defn de-dups [st]
+  (reverse (de-dups-helper st [])))
```

Comment:
reverse is only needed because you use `cons` an lists to build the accumulator, with vectors and `conj` reverse won't be necessary.